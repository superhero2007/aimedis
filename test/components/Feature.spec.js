import React from 'react';
import { mount } from 'enzyme';

import Feature from 'components/Feature';

function setup() {
  return mount(<Feature />);
}

describe('Feature', () => {
  const wrapper = setup();

  it('should be a StatelessComponent', () => {
    expect(wrapper.instance()).toBeNull();
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
