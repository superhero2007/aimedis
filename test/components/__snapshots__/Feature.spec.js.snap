// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feature should render properly 1`] = `"<div class=\\"app__feature\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__feature__title\\">Discover the best features for you.</div><div class=\\"app__feature__content\\"><div class=\\"app__get__early__access\\"><input class=\\"app__get__early__access__input\\" type=\\"text\\" placeholder=\\"Your Email Address\\"><button type=\\"button\\" class=\\"app__get__early__access__button btn btn__primary\\">Get <span class=\\"d-none d-lg-inline\\">Early </span>Access</button></div></div></div></div>"`;
